pkg/testing/runner/runner.go

```diff
@@ -230,7 +230,7 @@ func (r *Runner) runMachines(ctx context.Context, sshAuth ssh.AuthMethod, repoAr
 
 // runMachine runs the batch on the machine.
 func (r *Runner) runMachine(ctx context.Context, sshAuth ssh.AuthMethod, logger Logger, repoArchive string, batch LayoutBatch, machine OGCMachine) (OSRunnerResult, error) {
-	logger.Logf("Starting SSH connection to %s", machine.PublicIP)
+	logger.Logf("Starting SSH; auth in .ogc-cache; ip=%s, username=%s", machine.PublicIP, machine.Layout.Username)
```

Comment:
Why not just format this as a ssh connection line?

```
ssh -i .ogc-cache/id_rsa username@ip
```

Reply:
Ah good idea!